internal/provider/indices_datasource.go

```diff
@@ -372,13 +470,23 @@ func (d *indicesDataSource) Read(ctx context.Context, req datasource.ReadRequest
 			Type:                         types.StringValue(indexDetail.Type),
 			VectorNumericType:            types.StringValue(indexDetail.VectorNumericType),
 			Model:                        types.StringValue(indexDetail.Model),
-			NormalizeEmbeddings:          types.BoolValue(indexDetail.NormalizeEmbeddings),
+			ModelProperties: ModelPropertiesModel{
+				Name:             types.StringValue(indexDetail.ModelProperties.Name),
+				Dimensions:       types.Int64Value(indexDetail.ModelProperties.Dimensions),
+				Type:             types.StringValue(indexDetail.ModelProperties.Type),
+				Tokens:           types.Int64Value(indexDetail.ModelProperties.Tokens),
+				ModelLocation:    convertModelLocation(indexDetail.ModelProperties.ModelLocation),
+				Url:              types.StringValue(indexDetail.ModelProperties.Url),
+				TrustRemoteCode:  types.BoolValue(indexDetail.ModelProperties.TrustRemoteCode),
+				IsMarqtunedModel: types.BoolValue(indexDetail.ModelProperties.IsMarqtunedModel),
+			},
+			NormalizeEmbeddings: types.BoolValue(indexDetail.NormalizeEmbeddings),
 			TextPreprocessing: TextPreprocessingModel{
 				SplitLength:  types.StringValue(fmt.Sprintf("%d", indexDetail.TextPreprocessing.SplitLength)),
 				SplitMethod:  types.StringValue(indexDetail.TextPreprocessing.SplitMethod),
 				SplitOverlap: types.StringValue(fmt.Sprintf("%d", indexDetail.TextPreprocessing.SplitOverlap)),
 			},
-			//ImagePreprocessing: types.ObjectValue(map[string]interface{}, indexDetail.ImagePreprocessing),
+			// ImagePreprocessing
```

Comment:
Curious why we don't include this?

Reply:
Currently the cloud api does not return image_preprocessing (as well as audio and video) in the list index endpoint, so we might need to bring this up with dataplane team